client/signup/steps/domains/index.jsx

```diff
@@ -277,7 +277,20 @@ export class RenderDomainsStep extends Component {
 			suggestion?.is_premium
 		);
 		await this.props.saveSignupStep( stepData );
-		await this.submitWithDomain( { signupDomainOrigin, position } );
+
+		if ( shouldUseMultipleDomainsInCart( this.props.flowName ) && suggestion ) {
+			this.handleDomainToDomainCart();
```

Comment:
Since `handleDomainToDomainCart` doesn't need to be inside `submitWithDomain`, I removed it from there, so it's clear that for multi-domains we're NOT submitting with domain at this stage.

Reply:
```suggestion
		if ( shouldUseMultipleDomainsInCart( this.props.flowName ) && suggestion ) {
			await this.handleDomainToDomainCart();
```

Should we use await here? 🤔